compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala

```diff
@@ -196,6 +196,7 @@ enum ErrorMessageID(val isActive: Boolean = true) extends java.lang.Enum[ErrorMe
   case AmbiguousExtensionMethodID // errorNumber 180
   case UnqualifiedCallToAnyRefMethodID // errorNumber: 181
   case NotConstantID // errorNumber: 182
+  case ExtensionNullifiedByMemberID // errorNumber: 183
```

Comment:
I repaired the conflict, but it seems to me that MiMA should be enlisted to check that `errorNumber` is fixed once committed.

Reply:
I repaired it again. Happy 2024.